mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -44,7 +44,7 @@ private[ml] trait HasRegParam extends Params {
 private[ml] trait HasMaxIter extends Params {
 
   /**
-   * Param for maximum number of iterations (&gt;= 0).
+   * Param for maximum number of iterations (&gt;= 0). (default = 20)
```

Comment:
I don't see that this has a global default of 20. Where is that coming from?

Reply:
I think the default value (20) comes from https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala#L498 Is it wrong?

Reply:
That does not set a global default, no.

Reply:
I see. Sorry for my misunderstanding. I will close this.